src/main/features/response/routes/response-type.ts

```diff
@@ -36,9 +35,13 @@ export default express.Router()
         await DraftService.save(user.responseDraft, user.bearerToken)
 
         if (ResponseType.OWE_NONE === form.model.type) {
```

Comment:
Using `switch` here makes more sense, it should also make code a bit nicer / easier to read.